CONFIGURATION.md

```diff
@@ -11,6 +11,33 @@ Here are some Stargate-relevant property groups that are necessary for correct s
 * `quarkus.grpc.clients.bridge` - property group for defining the Bridge gRPC client (see [gRPC Client configuration](https://quarkus.io/guides/grpc-service-consumption#client-configuration) for all options)
 * `quarkus.cache.caffeine.keyspace-cache` - property group  for defining the keyspace cache used by [SchemaManager](../sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/schema/SchemaManager.java) (see [Caffeine cache configuration](https://quarkus.io/guides/cache#caffeine-configuration-properties) for all options)
 
+
+## Document limits configuration
+*Configuration for document limits, defined by [DocumentLimitsConfig.java](src/main/java/io/stargate/sgv2/jsonapi/config/DocumentLimitsConfig.java).*
+
+| Property                                                 | Type  | Default     | Description                                                                                                 |
+|----------------------------------------------------------|-------|-------------|-------------------------------------------------------------------------------------------------------------|
+| `stargate.jsonapi.document.limits.max-size`              | `int` | `1_000_000` | The maximum size of a single document in characters. Defaults to 1 million characters or approximately 1MB. |
```

Comment:
hmm not sure if this should be `stargate.jsonapi.document.limits`, maybe if we would also have `stargate.jsonapi.operations.limits`? Otherwise, lets just call this `stargate.jsonapi.document`?

Reply:
To me reference to "document" makes sense as it limits size of the stored document? Not sure why there should be reference to "operations".

Reply:
I think all similar limit settings should use same naming pattern: if this is true, all 3 look fine. (2) would be bit more compact so that sounds good if you want to change them.